### ISSUE TITLE
Inventory status_changed websocket messages clear in-progress creation form

### ISSUE THREAD

**Issue by AlanCoding** [ansible/awx#4029]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
User is interrupted filling out inventory form if unrelated action happens to some other inventory.

##### ENVIRONMENT
* AWX version: 4.0.0
* AWX install method: openshift, minishift, docker on linux, docker for mac, boot2docker
* Ansible version:  X.Y.Z
* Operating System:
* Web Browser:

##### STEPS TO REPRODUCE

Begin a script running in the background which makes API calls to the server to create inventories, delete inventories, and run jobs.

While this script is running, try to create a new inventory, filling out the form.

##### EXPECTED RESULTS

I expect that I will never lose an in-progress form during the course of normal use of AWX

##### ACTUAL RESULTS

The form gets reset when some form of websocket traffic comes in.

![alan_3k](https://user-images.githubusercontent.com/1385596/59201735-10ba9100-8b69-11e9-93e7-c4222e9dc110.gif)

Sorry it's not clear from the crop, but some other non-empty inventory is created and deleted within the time the video is running.

**Comment by ryanpetrello** [comment]
cc @mabashian

**Comment by mabashian** [comment]
Note that the same thing happens on the projects list/form

**Comment by mabashian** [comment]
Actually - I think this happens on every list/form combo where the list contents are refreshed via websockets.

**Comment by mabashian** [comment]
OK so the original issue is pretty clear: If you are adding/editing an inventory when an inventory is deleted then the UI will basically refresh the page which clears out/resets the form.  I've got a fix queued up for that which I'll link shortly.

While investigating this I thought I was coming across the same behavior on the projects and templates lists.  I had two tabs open (one on the projects list and one on the project add form).  If I triggered an event (lets say `sync`) on the projects list and then switched back to the add project form tab the form is wiped.  The behavior is similar to the original bug but I think the reasoning is different.  I suspect that this has something to do with the changes we made to not process websocket events that come in while the tab is in the background.  When the tab comes back into focus it seems like the entire state is reloaded (which would wipe the form).
